ui/prometheus-plugin/src/plugins/prometheus-time-series-query/PrometheusTimeSeriesQueryEditor.tsx

```diff
@@ -59,41 +66,46 @@ export function PrometheusTimeSeriesQueryEditor(props: PrometheusTimeSeriesQuery
     throw new Error('Got unexpected non-Prometheus datasource selector');
   };
 
-  if (isExplore) {
-    return (
-      <ExplorePrometheusTimeSeriesQueryEditor
-        selectedDatasource={selectedDatasource}
-        handleDatasourceChange={handleDatasourceChange}
-        promURL={promURL}
-        query={query}
-        handleQueryChange={handleQueryChange}
-        handleQueryBlur={handleQueryBlur}
-        format={format}
-        handleFormatChange={handleFormatChange}
-        handleFormatBlur={handleFormatBlur}
-        minStepPlaceholder={minStepPlaceholder}
-        minStep={minStep}
-        handleMinStepChange={handleMinStepChange}
-        handleMinStepBlur={handleMinStepBlur}
-      />
-    );
-  }
-
   return (
-    <DashboardPrometheusTimeSeriesQueryEditor
-      selectedDatasource={selectedDatasource}
-      handleDatasourceChange={handleDatasourceChange}
-      promURL={promURL}
-      query={query}
-      handleQueryChange={handleQueryChange}
-      handleQueryBlur={handleQueryBlur}
-      format={format}
-      handleFormatChange={handleFormatChange}
-      handleFormatBlur={handleFormatBlur}
-      minStepPlaceholder={minStepPlaceholder}
-      minStep={minStep}
-      handleMinStepChange={handleMinStepChange}
-      handleMinStepBlur={handleMinStepBlur}
-    />
+    <Stack spacing={2}>
+      <FormControl margin="dense" fullWidth={false}>
+        {/* TODO: How do we ensure unique ID values if there are multiple of these? Can we use React 18 useId and
+                maintain 17 compatibility somehow with a polyfill/shim? */}
+        <InputLabel id="prom-datasource-label">Prometheus Datasource</InputLabel>
```

Comment:
Couldn't we use the datasource name?
```suggestion
        <InputLabel id={`prom-datasource-label-${datasource.name}`">Prometheus Datasource</InputLabel>
```

Reply:
Updated with a variable computed this way 
```ts
const datasourceSelectLabelID = `prom-datasource-label-${selectedDatasource.name || 'default'}`;
```